broadcast/internal/arc/arc_policy_quote.go

```diff
@@ -45,8 +45,8 @@ func getPolicyQuote(ctx context.Context, arc *ArcClient) (*broadcast.PolicyQuote
 		nil,
 	)
 
-	if arc.headers != nil {
-		pld.Headers = arc.headers
+	if arc.deploymentID != "" {
+		pld.AddHeader("XDeployment-ID", arc.deploymentID)
```

Comment:
Consider putting the "XDeployment-ID" header key to a const global variable, because the string is re-used several times.

Reply:
moved to const

Reply:
Nice 👍